.eslintrc.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Not sure if this config. adds default eslint recommended rules (https://eslint.org/docs/rules/)
You might like to add 
  ```
"extends": [
    "eslint:recommended"
  ]
```

Reply:
Hi Sachin, I definitely intend to add this in the near future, but I'm leaving it out of this PR - The reason being we intend to migrate to typescript and ES6 syntax soon and that will make it a lot easier to write code that passes a ruleset like `eslint:recommended` (or even `airbnb-base`).

Reply:
Yeah, makes sense ( I would also suggest airbnb styling for default linting rules)